.github/workflows/tests.yml

```diff
@@ -2,6 +2,8 @@ name: Tests
 
 on:
   push:
+    branches-ignore:
+      - "dependabot/**"
```

Comment:
Any reason to skip CI for dependabot branches?

Reply:
Because they get submitted as PRs, they already get built for the PR. This avoids running the same build twice

Reply:
Isn't it the same for other PRs?

Reply:
Yes, PRs that you create would also double build. The alternate that some folks use would be:
```
push:
  branches:
    - master
```
So it only builds when the default branch pushes. I depends if you want to see CI builds while you work on your own stuff though

Reply:
After investigation, I prefer this strategy. Thanks for pointing this out :)